app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | Exception:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("You are not vaccinated")

```

Comment:
Here you are comparing a variable to `None` or `0`. According to the checklist, you should avoid such comparisons if there is no need. You can just write `if not visitor.get('vaccine'):`.